roles/kubernetes/control-plane/tasks/kubeadm-etcd.yml

```diff
@@ -16,3 +16,10 @@
   import_role:
     name: etcdctl
   when: etcd_deployment_type == "kubeadm"
+
+- name: Set ownership for etcd data directory
+  file:
+    path: "{{ etcd_data_dir }}"
+    owner: etcd
```

Comment:
Don't we need to specify "{{ etcd_owner }}" instead of a static `etcd`?

Reply:
Yes, probably is better.

Reply:
Sorry for that I could not get the point, which is better?

Reply:
Sorry for misunderstanding. I meant, probably is metter to add the variable `{{ etcd_owner }}` instead of the static `etcd`.

Reply:
ah, OK.
Then I will add approval again after updating the above place.